content/en/docs/application-store/app-lifecycle-management.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "App Lifecycle Management"
+keywords: 'kubernetes, kubesphere, app-store'
+description: 'App Lifecycle Management'
+
+
+weight: 2240
+---
+
+KubeSphere integrates open source [OpenPitrix](https://github.com/openpitrix/openpitrix) to set up app store and app repository services which provide full lifecycle of application management. Application Store supports three kinds of application deployment as follows:
```

Comment:
`KubeSphere integrates open-source project [OpenPitrix](https://github.com/openpitrix/openpitrix) to set up the App Store and which provide the full lifecycle of application management. App Store supports two kinds of application deployment as follows:`

Reply:
got it